firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -345,7 +345,9 @@ class BrowserViewController: UIViewController,
         }
 
         ratingPromptManager.updateData()
-        ratingPromptManager.showRatingPromptIfNeeded()
+        if featureFlags.isFeatureEnabled(.ratingPromptFeature, checking: .buildOnly) {
+            ratingPromptManager.showRatingPromptIfNeeded()
+        }
```

Comment:
do we also want to add `ratingPromptManager.updateData()` nested in the feature flag as well? 
Even better, if this is called only here, maybe we can make these two methods private and call one method `ratingPromptManager.handleRatingPromptFlow` or a better name 🤣

Reply:
I considered putting the `updateData` in the feature flag, but this `updateData` is not doing much behind the scenes. Also the upside is that we are accurately tracking if the user crashed in the last 3 days (so we don't ask them to rate us if they crashed). If we don't `updateData`, then the scenario where a user crashed in the last 3 days and we show them the rating prompt could happen.

Reply:
ah interesting, thanks for the context! Hmm in that case, it feels a bit weird that we're updating our crash data in the rating prompt manager as opposed to keeping it decoupled then.

Reply:
I added a "CrashTracker" with my latest update, let me know if that works

Reply:
nice! works great :) thanks for the quick change!